backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/sort/impl/ElasticsearchScoreSortBuilder.java

```diff
@@ -28,4 +29,9 @@ public void doBuildAndAddTo(ElasticsearchSearchSortCollector collector, JsonObje
 			collector.collectSort( outerObject );
 		}
 	}
+
+	@Override
+	public void multi(SortMultiFunc multi) {
+		throw new UnsupportedOperationException( "Not supported yet." ); //To change body of generated methods, choose Tools | Templates.
```

Comment:
This looks incomplete?

Reply:
I suspect there is a solution. The more that I used the code from elasticsearch. It is difficult for me to solve the problem here because I do not use elasticsearch. In such cases, I hope that the right person will complete this code. The exception gives this information.

Reply:
Changed, I set the parameter in the building material. The multi value is set, similarly in the numeric range and terms aggregation. In Apache Lucene this has been resolved, but in Elasticsearch it has no effect. But it can be used if there is a similar option in Elasticsearch.